LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
```

Comment:
LICENSE should be committed as a separate commit.

Reply:
There exist another LICENSE file, this is one kind of LICENSE format that mentor tell us